### ISSUE TITLE
Weapons missing?

### ISSUE THREAD

**Issue by Alexander-Krause** [frenos/btl44#15]
I didn't change my docker-compose file, but since the new update I don't have any weapons available on the servers.

Do you have the same problem?

**Comment by frenos** [comment]
We run two arcade servers that work fine after the update but we run them with default settings, setting admins and name with environment vars.
You could diff the new defaultGame and loadout against your custom ones.

**Comment by Alexander-Krause** [comment]
You are right, something in the DefaultGame.ini changed. Therefore, my old mounted .ini failed the application. As a result, no weapons were available.

**Comment by Alexander-Krause-Glau** [comment]
Hi,

unfortunately I misjudged, therefore I am still experiencing this issue. The following command shows a single container I am running:

`docker run -d --name btlserver -e 'PUBLICIP=XXX' -e 'PORT=XXX' -e 'SERVERNAME="XXX"' -e 'PLAYMODE=unranked' -e 'ADMINSTEAMID=XXX' -e 'STARTTYPE=PlayerCount' -e 'REQUIREDPLAYERS=2' frenos/btl44`

Procedure:
1. Connect to server (new container -> everything should be fine, you can play the map to the end)
2. Change to any map with Server.ChangeMap X BOMB
3. Server loads map and you will automatically join in spectator mode
4. Join team with "m", you are now stucked with wiggling hands and cannot choose any weapon

Can you reproduce the problem?

**Comment by frenos** [comment]
Yes, I can finally reproduce this on my server. It seems to be happening if there is only one person connected while switching?
At least this did not happen when we changed maps with 5-6 ppl on the server.

Will test this on a native installation now, reporting back.

**Comment by Alexander-Krause-Glau** [comment]
@frenos Thanks. I indeed tested only with 1 person on the server. Looking forward to your results.

**Comment by frenos** [comment]
We tested it a lot the last days, here are our findings:
- It also happens with a server directly installed via steamcmd (fresh debian)
- It does not matter which gamemode you switch to
- It happens always to the first person connected and just involves that person, not the whole server
- If the bugged person gets killed it is fixed
- It also happens with "official" unranked servers

So all in all it seems to be a bug in the server and not related to the container (whew :)).

**Comment by Alexander-Krause-Glau** [comment]
Nice work, thank you !
